java/core/src/main/scala/io/pdal/PackedPoints.scala

```diff
@@ -0,0 +1,66 @@
+package io.pdal
+
+import java.nio.ByteBuffer
+import java.util
+
+import scala.collection.JavaConversions._
+
+/**
+  * PackedPoints abstraction to work with packed point(s) in JVM memory.
+  * SizedDimType contains size and offset for a particular packed point with the current set of dims.
+  **/
+case class PackedPoints(bytes: Array[Byte], dimTypes: util.Map[String, SizedDimType], metadata: String = "", schema: String = "") {
```

Comment:
Great, perhaps it makes sense to try and push as much relevant functionality into class so others could use it. That functionality move could maybe happen as separate PRs after we have a working DEM conversion, to test on our side and not complicate development; but if we rename this PointCloud we'll have a naming conflict, which we can get around using the extension methods, or absorbing some of more simple methods into here.